package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "leaderboard-task",
+  "version": "1.0.0",
+  "description": "",
+  "main": "webpack.config.js",
+  "scripts": {
+    "build": "webpack",
+    "start": "webpack serve --open",
+    "watch": "webpack --watch",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "eslint": "eslint .",
+    "stylelint": "stylelint \"**/*.{css,scss}\""
+  },
```

Comment:
- [ ] Please, add the next option here `"dev": "webpack --mode development",`.

Reply:
Applied changes successfully :+1: